Dockerfile.local

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.19-bullseye AS base
+
+ENV GOCACHE=/go/.go/cache GOPATH=/go/.go/path TZ=Europe/London
+
+# RUN GOBIN=/bin go get github.com/cespare/reflex
```

Comment:
This comment seems a little too much like code. Something like this might be better: 
```suggestion
# Install github.com/cespare/reflex
```